bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -49,8 +50,8 @@ public static CloudBigtableScanConfiguration fromCBTOptions(CloudBigtableOptions
   }
 
   /**
-   * Converts a {@link CloudBigtableOptions} object to a {@link CloudBigtableScanConfiguration},
-   * with the addition of a {@link Scan} that filters the table.
+   * Converts a {@link CloudBigtableOptions} object to a {@link CloudBigtableScanConfiguration}
+   * object with the addition of a {@link Scan} that filters the table.
```

Comment:
what about  "...CloudBigtableScanConfiguration that will perform the specified Scan on the table." ?